regolith/helpers/l_contactshelper.py

```diff
@@ -0,0 +1,136 @@
+"""Helper for finding and listing contacts from the contacts.yml database.
+Prints name, institution, and email (if applicable) of the contact.
+"""
+import datetime 
+import dateutil
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    fuzzy_retrieval,
+)
+
+TARGET_COLL = "contacts"
+HELPER_TARGET = "l_contacts"
+
+def subparser(subpi):
+    subpi.add_argument("-n", "--name", help='finds contacts based off a list of names or name fragments', nargs = '+')
+    subpi.add_argument("-i", "--inst", help='finds contacts based off an institution or an institution fragment')
+    subpi.add_argument("-d", "--date", help='finds contacts based off a date in ISO format (YYYY-MM-DD) with a default range of 4 months centered around the date (d +/- 2 month)')
+    subpi.add_argument("-r", "--range", help='optional argument that sets range (in months) centered around date d (d +/- r/2)')
+    subpi.add_argument("-no", "--notes", help='finds contacts based off notes or miscellaneous information about the contact', nargs = '+')
+    return subpi
+
+def is_current(thing, now=None):
```

Comment:
remove.   You will import this from ``dates.py``.  Sorry about this duplication of effort....Michael's PR should have put it there by now, but I am not sure if is merged yet.

Reply:
I actually copied this from dates.py, but I had to change the function because it didn't align with my code. I'm using dateutil to create the range for my dictionary that contains begin_date and end_date, so my dictionary's values are datetime objects. Meanwhile, is_current calls the get_dates function from dates.py, which requires the dictionary to not have datetime objects, but strings instead. So I copied the exact function, but just changed the line that calls the get_dates function to not call it instead. The alternative would be to manually convert the dictionary with datetime objects to a dictionary with string objects, only for the function to convert it right back (so doing "double the work" in a sense, which could slow down the program). That said, I'm open to either way, so please let me know which one you would prefer?